src/components/application-future-cost/cost-estimate-content.js

```diff
@@ -1,17 +1,18 @@
 import applicationCost from '../../models/application-cost';
 import PropTypes from 'prop-types';
 import React from 'react';
+import moment from 'moment';
 
 export const CostEstimateContent = ({ applicationCost }) => {
   if (!applicationCost) {
     return <table></table>;
   }
 
   function getPeriod() {
-    var today = new Date();
-    var nextMonth = new Date();
-    nextMonth.setDate(nextMonth.getDate() + 30);
-    return `${today.toLocaleDateString()} - ${nextMonth.toLocaleDateString()}`;
+    var today = moment();
+    var nextMonth = moment(today).add(30, 'days');
+    return `${today.format('DD MMM YYYY')}
+    - ${nextMonth.format('DD MMM YYYY')}`;
```

Comment:
Suggest to keep format in `src/utils/datetime.js`

Reply:
Implemented your suggesation